### PR TITLE
Internal updates

### PR DESCRIPTION
**Pull request by scoland**
An update to this pull request https://github.com/amazon-connect/amazon-connect-streams/pull/334 with some minor changes.

* Removed Config file
* Fixed a typo
* Bumped the minor release version to 1.6

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jolim24601 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,17 +127,18 @@ everything setup correctly and that you will be able to listen for events.
...
   </head>
   <!-- Add the call to init() as an onload so it will only run once the page is loaded -->
   <body onload="init()">
-    <div id="containerDiv" style="width: 400px;height: 800px;"></div>
+    <div id="container-div" style="width: 400px;height: 800px;"></div>
     <script type="text/javascript">
+      var containerDiv = document.getElementById("container-div");
       var instanceURL = "https://my-instance-domain.awsapps.com/connect/ccp-v2/";
-      // initialize the streams api
+      // initialise the streams api
```
Comment (h3adache) [inline README.md:141]: typo

File: src/api.js [hunk 2]
```diff
@@ -254,6 +254,10 @@
     connect.core.getUpstream().onUpstream(connect.AgentEvents.MUTE_TOGGLE, f);
   };
 
+  Agent.prototype.onLocalMediaStreamCreated = function (f) {
```
Comment (h3adache) [inline src/api.js:257]: We are missing documentation for this still

File: src/api.js [hunk 3]
```diff
@@ -659,13 +668,6 @@
       });
   };
 
-  Contact.prototype.destroy = function (callbacks) {
```
Comment (h3adache) [inline src/api.js:662]: We shouldn't remove this. Deprecate this as in the documentation.  
noop and log error (or warn) on use as @ctwomblyamzn suggested on #334

**Review by h3adache — CHANGES_REQUESTED** [review]

File: src/api.js (hunk 2 above)
Comment (scoland) [reply src/api.js:257]: Removing this for this release.

**Review by h3adache — APPROVED** [review]

**Review by ctwomblyamzn — APPROVED** [review]

**Pull request merged**